eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -68,28 +67,18 @@ Get-Content -Raw path/to/file.yml | CompatibleConvertFrom-Yaml
 #>
 function CompatibleConvertFrom-Yaml {
   param(
-    [Parameter(Mandatory=$true, ValueFromPipeline=$true)]
+    [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
     [string]$Content
   )
 
   if (!($Content)) {
     throw "Content to parse is a required input."
   }
 
-  # Initialize any variables or checks that need to be done once
-  $yqPresent = Get-Command 'yq' -ErrorAction SilentlyContinue
-  if (-not $yqPresent) {
-    . (Join-Path $PSScriptRoot PSModule-Helpers.ps1)
-    Install-ModuleIfNotInstalled -WhatIf:$false "powershell-yaml" "0.4.7" | Import-Module
-  }
+  . (Join-Path $PSScriptRoot PSModule-Helpers.ps1)
+  Install-ModuleIfNotInstalled -WhatIf:$false "powershell-yaml" "0.4.7" | Import-Module
```

Comment:
I think the other idea here was to replace this with below, so in the case where it's a no-op we don't waste time on `import-module`. I can take that on though if you have other stuff to do, wasn't sure which piece you were volunteering to do.

```
  InstallAndImport-ModuleIfNotInstalled -WhatIf:$false "powershell-yaml" "0.4.7"
```

Reply:
How can we call a function without importing it?

Reply:
Oh I see what you're saying. You're saying I should also be able to avoid `| Import-Module`.

Reply:
I think it would just be something like this, but in `PSModule-Helpers.ps1`. As implemented in this PR, the no-op check will still take a couple seconds each time because we are pipelining the already installed module to `Import-Module`.

```
function InstallAndImport-ModuleIfNotInstalled([string]$module, [string]$version) {
  if ($null -eq (moduleIsInstalled $module $version)) {
    Install-ModuleIfNotInstalled -WhatIf:$false $module $version | Import-Module
  } elseif (!(Get-Module -Name $module)) {
    Import-Module $module
  }
}   
```